### ISSUE TITLE
Raw files for CMU MOSEI dataset

### ISSUE THREAD

**Issue by naveenDevPyxeda** [A2Zadeh/CMU-MultimodalSDK#355]
Hi I am looking for the raw files of CMU MOSEI dataset. Such as raw files of audio, video and texts. I searched through the internet but i could not find anything.

Anyone know where I can find the raw files.? @A2Zadeh

**Comment by xylarshayu** [comment]
@naveenDevPyxeda Hi can you please share where you find the mosei.csv file? I have been unable to obtain it

**Comment by naveenDevPyxeda** [comment]
http://immortal.multicomp.cs.cmu.edu/timestamped_links/

**Comment by Jing-Jing-Guo** [comment]
Hi @naveenDevPyxeda ! I tried to extract raw data from the link above, but it didn't work.  It mentioned in README.md that they don't provide raw data, so I suppose we can't get raw data in the end.  ( I am trying to find raw data, too.  lol)

**Comment by Jing-Jing-Guo** [comment]
Indeed, however when you click on the links, it appears that no videos are available anymore.

**Comment by Jing-Jing-Guo** [comment]
Hi @naveenDevPyxeda ! Sorry, I just tried to open the links again, and they now work lol. I'm not sure why it appeared unavailable previously, but I'll try to get the videos again. Thank you.

**Comment by csDevin** [comment]
@Jing-Jing-Guo Hello, I found that the links to many videos in mosi.csv are invalid. Is there any way to get these invalid videos?

**Comment by Jing-Jing-Guo** [comment]
> @Jing-Jing-Guo Hello, I found that the links to many videos in mosi.csv are invalid, e.g., https://www.youtube.com/watch?v=LSi-o-IrDMs, https://youtube.com/watch?v=225768. (youtube shows ”this video isn't available anymore")
> Is there any way to get these invalid videos?

@csDevin Hi, I encountered the same issue, but I just ignored it lol. I solely obtained the accessible videos for a quick test. Sorry for not being able to solve this problem yet :(

**Comment by Jing-Jing-Guo** [comment]
> @Jing-Jing-Guo Hello, I found that the links to many videos in mosi.csv are invalid, e.g., https://www.youtube.com/watch?v=LSi-o-IrDMs, https://youtube.com/watch?v=225768. (youtube shows ”this video isn't available anymore")
> Is there any way to get these invalid videos?

@csDevin Hi, I encountered the same issue, but I just ignored it lol. I solely obtained the accessible videos for a quick test. Sorry for not being able to solve this problem yet :(
